syntax = "proto3";

package taraxa.vm.statedb;

option go_package = "grpc_go";

message VmId {
    bytes contractAddress = 1;
    string processId = 2;
}

message BytesMessage {
    bytes value = 1;
}

message BoolMessage {
    bool value = 1;
}

